data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml

```diff
@@ -24,8 +24,9 @@ terraform:
     hana_instancetype: "%PUBLIC_CLOUD_INSTANCE_TYPE%"
 
 ansible:
-  hana_urls:
+  az_storage_account_name: "%HANA_ACCOUNT%"
+  az_container_name:  "%HANA_CONTAINER%"
+  hana_media:
```

Comment:
Example of a resulting hana_media.yaml is https://openqa.suse.de/tests/10945955/file/qesap_terraform-hana_media.yaml

It is not ok. Please compare it with http://openqaworker15.qa.suse.cz/tests/146995/file/configure-hana_media.yaml that is using https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/sles4sap/qe_sap_deployment/qesap_aws.yaml

`az_sas_token` is missing

Reply:
Yes, I noticed this. But test module can pass so I did not add "az_sas_token". Why no "az_sas_token" test module still can pass?

Reply:
Added "az_sas_token".

Reply:
> Why no "az_sas_token" test module still can pass?

As mr_test is not calling Ansible at all